### PR TITLE
Revamp README.md

### PR DESCRIPTION
**Pull request by QazCetelic**
[README.md](https://github.com/QazCetelic/JetBrainsRunner/blob/master/README.md)

### CODE REVIEW

**Comment by QazCetelic** [comment]
Added changes from #21

File: README.md [hunk 1]
```diff
@@ -1,39 +1,87 @@
...
 
-Archlinux(Manjaro):  
-`sudo pacman -S cmake extra-cmake-modules libnotify kcmutils kio`
+### 2 KRunner Plugin
+#### 2.1 Get Files
 
-The easiest way to install is:  
-`curl https://raw.githubusercontent.com/alex1701c/JetBrainsRunner/master/install.sh | bash`  
+<details>
+<summary>Option A: Easy oneliner method</summary>
```
Comment (alex1701c) [inline README.md:80]: This requires admin privileges too

File: README.md [hunk 2]
```diff
@@ -1,39 +1,87 @@
...
-New Feature:
-Now you can pin your projects by dragging the KRunner results to the taskbar or desktop.
+- The search suggestions and the application ⇒ config file mapping can be customized.
+- File watchers are implemented, these reload the configuration when new projects get added/removed.
+- You can pin your projects by dragging the KRunner results to the taskbar or desktop.
 
-To install the plugin you can visit out the release page(https://github.com/alex1701c/JetBrainsRunner/releases)
-or build from source as explained below.
+<details>
+<summary><b>Screenshots</b></summary>
```
Comment (alex1701c) [inline README.md:18]: I am wondering if it is a good idea to all screenshots by default.

For the config dialog I agree, that might not be important when one gets an initial impression of what this plugins does. However I think providing a simple screenshot how the matches will be displayed helps the user. (That is how I felt when first scrolling through the available KRunner plugins on github a few years ago ;))

**Comment by alex1701c** [comment]
I think this is really nice, thanks a lot!

File: README.md [hunk 3]
```diff
@@ -42,43 +90,42 @@ cd build
...
+
+#### Result Formatting
+You can change the displayed text for the run options using the following rules.
+- *%APPNAME*: application name
+- *%APP*: shortened application name
+- *%PROJECT*: project name
+- *%DIR*: full directory path
+
+#### Manually Adding IDE's
+If the plugin has issues finding the installed applications or finding the correct config file you can specify them manually, you are welcome to create a pull request and add it to the [default configuration](https://github.com/alex1701c/JetBrainsRunner/blob/master/src/JetbrainsApplication.cpp#L137).
```
Comment (alex1701c) [inline README.md]: I realized that this is out of sync with a change I did some time ago.

Now the file paths are no longer hardcoded. If one encounters problems that require manual configuration, it is be best to create an issue here on Github.

File: README.md [hunk 4]
```diff
@@ -1,39 +1,93 @@
...
+### 2 KRunner Plugin
+#### 2.1 Get Files
 
-Archlinux(Manjaro):  
-`sudo pacman -S cmake extra-cmake-modules libnotify kcmutils kio`
+<details>
+<summary>Option A: Easy oneliner method</summary>
+
+```
+sudo curl https://raw.githubusercontent.com/alex1701c/JetBrainsRunner/master/install.sh | bash
```
Comment (alex1701c) [inline README.md]: The install.sh script will ask for admin privileges when installing. The download, configuring and compiling does not require those privileges.

**Comment by alex1701c** [comment]
Thanks

**Pull request merged**